translation/ja/patterns/service-vs-library.md

```diff
@@ -50,25 +50,25 @@ DevOps環境のチームは、サービスのダウンタイムに対応する
 
 ## その他の情報
 
-このパターンに関連するものとして、上記を解決するために異なるアプローチをとる[30日の保証期間](30-day-warranty.md) パターンがあります。
+このパターンに関連するものとして、上記を解決するために異なるアプローチをとる[30 日の保証期間](30-day-warranty.md) パターンがあります。
 
 ## 事例
 
-* ユーロスペース AG
-* Flutter Entertainment: [Flutter インナーソースの応用](https://innersource.flutter.com/start/setup-ci/)には、チーム横断的にコントリビュートする共有コードの"サービスリポジトリ"と、共有リリース成果物をビルドして公開するためのCIパイプラインがあります。各チームは、独自のデプロイメントを定義する"デプロイメント設定"リポジトリを持っています。これは、さまざまな規制要件、サービスおよびインシデント管理の実践、ビジネスの各領域におけるインフラストラクチャのスキルセットによって異なる運用がされています。
+- ユーロスペース AG
+- Flutter Entertainment: [Flutter インナーソースの応用](https://innersource.flutter.com/sdlc/)には、チーム横断的にコントリビュートする共有コードの"サービスリポジトリ"と、共有リリース成果物をビルドして公開するための CI パイプラインがあります。各チームは、独自のデプロイメントを定義する"デプロイメント設定"リポジトリを持っています。これは、さまざまな規制要件、サービスおよびインシデント管理の実践、ビジネスの各領域におけるインフラストラクチャのスキルセットによって異なる運用がされています。
```

Comment:
Looks like new whitespace is introduced here between Japanese and English characters.

Did you introduce these on purpose, or is this something that maybe your editor did automatically?

Reply:
yes, IDE auto-formatter. Argh, reverted.